tests/secc/states/test_messages.py

```diff
@@ -115,3 +123,53 @@ def get_dummy_v2g_message_authorization_req():
         header=MessageHeader(session_id=MOCK_SESSION_ID),
         body=Body(authorization_req=authorization_req),
     )
+
+
+def get_charge_parameter_discovery_req_message_departure_time_one_hour():
+    e_amount = PVEAmount(multiplier=0, value=60, unit=UnitSymbol.WATT_HOURS)
+    ev_max_voltage = PVEVMaxVoltage(multiplier=0, value=400, unit=UnitSymbol.VOLTAGE)
+    ev_max_current = PVEVMaxCurrent(multiplier=-3, value=32000, unit=UnitSymbol.AMPERE)
+    ev_min_current = PVEVMinCurrent(multiplier=0, value=10, unit=UnitSymbol.AMPERE)
+    one_hour_in_seconds = 3
+    ac_charge_params = ACEVChargeParameter(
+        departure_time=one_hour_in_seconds,
+        e_amount=e_amount,
+        ev_max_voltage=ev_max_voltage,
+        ev_max_current=ev_max_current,
+        ev_min_current=ev_min_current,
+    )
+
+    charge_parameter_discovery_req = ChargeParameterDiscoveryReq(
+        requested_energy_mode=EnergyTransferModeEnum.AC_THREE_PHASE_CORE,
+        ac_ev_charge_parameter=ac_charge_params,
+        dc_ev_charge_parameter=None,
+    )
+
+    return V2GMessage(
+        header=MessageHeader(session_id=MOCK_SESSION_ID),
+        body=Body(charge_parameter_discovery_req=charge_parameter_discovery_req),
+    )
+
+
+def get_charge_parameter_discovery_req_message_no_departure_time():
+    e_amount = PVEAmount(multiplier=0, value=60, unit=UnitSymbol.WATT_HOURS)
+    ev_max_voltage = PVEVMaxVoltage(multiplier=0, value=400, unit=UnitSymbol.VOLTAGE)
```

Comment:
can you add these to a conftest.py file and add the following decorator on each method: "@pytest.fixture"

you can then use directly the return of that method into the test_ method you use. For example:

```python
@pytest.fixture
def get_charge_parameter_discovery_req_message_no_departure_time():
    ....
```

```python
async def test_charge_parameter_discovery_res_v2g2_761(self, discover_req_message_no_dep_time):
    # V2G2-761: If departure time was not provided, then SECC shall assume
    # that the EV intends to start charging without any delay
    charge_parameter_discovery = ChargeParameterDiscovery(self.comm_session)
    charge_parameter_discovery.process_message(
        message=discover_req_message_no_dep_time
    )
    assert (
        charge_parameter_discovery.next_msg.body.charge_parameter_discovery_res
        is not None
    )
```